### ISSUE TITLE
Forced Predictors defined at the Equation Pool level

### ISSUE THREAD

**Issue by tjrocha** [usbr/PyForecast#4]
@kevinfol - Looking for feedback on how you and/or your users feel about PN handling forced predictors. We can either Option-1)make another **_ForcedPredictor_** list at the **_EquationPool_**-level to store the **_PredictorIDs_** so users can make a distinction for which predictors are 'forced' into the algorithm or Option-2)we can make this a little more hidden by just storing a parallel array under the existing **_PredictorPool_** list to track which **_PredictorIDs_** should be forced at feature-selection time. 

Screenshot below shows what I'm thinking about. Option-1 will have a separate list of **_ForcedPredictors_** on the GUI or Option-2 will allow users to right-click on predictors and set them to _forced_.

![image](https://user-images.githubusercontent.com/8174126/52071596-933e9e80-2540-11e9-84ae-4df92045cc32.png)

**Comment by kevinfol** [comment]
I'd prefer Option 2. Perhaps it would be useful to mark those predictors that are being forced with a little Ⓕ Icon, for example, in PyForecast_GUI.py at the 'addDictToModel' function:

```python
def addDictToModel(self, model, dict_, ....):
    ...
                else:
                    if isinstance(key, datetime):
                        key = datetime.strftime(key, '%Y')
                    if key in dict_[...]['forcedPredictorArray']:
                         item = QtGui.QStandardItem(u'\u24BB' + str(key) + ': ' + str(values))
                    else:
                         item = QtGui.QStandardItem(str(key) + ': ' + str(values))
                    parentItem.appendRow(item)
```

**Comment by tjrocha** [comment]
Going with Option-2 then. Thanks!

**Comment by tjrocha** [comment]
Implemented with PR #9. Will close this issue once PR is accepted.
